assets/templates/aws.ec2_logs/schema-b/configs.yml

```diff
@@ -0,0 +1,18 @@
+- name: process.name
+  enum: ["journal", "kernel", "systemd"]
+- name: agent.id
+  value: "da6cb4c8-c84c-4c5f-97c7-f8586a098af4"
+- name: cloud.region
+  enum: ["us-east-1", "us-east-2", "us-west-1", "us-west-2", "ap-south-1", "ap-northeast-3", "ap-northeast-2", "ap-southeast-1", "ap-southeast-2", "ap-northeast-1", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3", "eu-north-1", "sa-east-1", "af-south-1", "ap-east-1", "ap-south-2", "ap-southeast-3", "eu-south-2", "eu-central-2", "me-south-1", "me-central-1"]
```

Comment:
nit: if we don't give the same cardinality to `cloud.region`, the same `host.name` could end up in different regions

Reply:
yes, 1/100 is correct